client/auth/index.js

```diff
@@ -10,8 +10,9 @@ import config from 'config';
 import controller from './controller';
 
 export default () => {
+	page( '/oauth-login', controller.oauthLogin );
```

Comment:
Agreed, it's a temporary fix though and this controller is already redirecting in some cases so I found it was the right place to add this condition.
Ok for removing oauth in a follow up PR (even though it's quite a shame that we're moving away from it...)